databricks/koalas/frame.py

```diff
@@ -3059,8 +3125,9 @@ class  locomotion
         sdf = self._internal.spark_frame.filter(reduce(lambda x, y: x & y, rows)).select(scols)
 
         if len(key) == len(self._internal.index_spark_columns):
-            result = first_series(DataFrame(InternalFrame(spark_frame=sdf, index_map=None)).T)
-            result.name = key
+            result = first_series(
+                DataFrame(InternalFrame(spark_frame=sdf, index_map=None)).T
+            ).rename(key)
```

Comment:
@ueshin, out of curiosity, was it changed only because of the style or because `ser.name = key` doesn't work?

Reply:
At that time I was working around here, `ser.name = key` didn't work.
We can revert it but I thinks `.rename(key)` is more natural here.